bzt/modules/jmeter.py

```diff
@@ -1639,6 +1662,12 @@ def __parse_request(self, req):
                 raise ValueError("`do` field is mandatory for loop blocks")
             do_requests = self.__parse_requests(do_block)
             return LoopBlock(loops, do_requests, req)
+        elif 'transaction' in req:
+            name = req.get('transaction')
+            gen_parent_sample = req.get('generate-parent-sample', False)
```

Comment:
I'd remove this option, it has no good use. The only way to have transactions is to have parent sample. Otherwise average statistics shows useless info. At least I wouldn't mention it in the docs so much.